app/models/notification.rb

```diff
@@ -4,11 +4,12 @@
 # This class' responsibility is, given a notification blueprint, to determine
 # who are the notification recipients
 class Notification < ApplicationRecord
-  AUDIENCE_USER     = 'user'
-  AUDIENCE_GROUP    = 'usergroup'
-  AUDIENCE_TAXONOMY = 'taxonomy'
-  AUDIENCE_GLOBAL   = 'global'
-  AUDIENCE_ADMIN    = 'admin'
+  AUDIENCE_USER       = 'user'
+  AUDIENCE_GROUP      = 'usergroup'
+  AUDIENCE_TAXONOMY   = 'taxonomy'
+  AUDIENCE_GLOBAL     = 'global'
+  AUDIENCE_ADMIN      = 'admin'
+  AUDIENCE_HOST_OWNER = 'host_owner'
```

Comment:
does it make sense to keep both host_owner and user/usergroup? I'm trying to think of usage cases I'll notify a group and I can't (besides the host context).